projects/plugins/crm/js/jpcrm-admin-funnel.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Draws a simple funnel
+ * 
+ * Inspired in part by https://codepen.io/tylerlwsmith/pen/RwNZKgp
+ * 
+ * @param funnel_data array of funnel data (count, backfill_count, contact_status, and link)
+ * @element HTML element to turn into a funnel
+ **/
+
+function jpcrm_build_funnel(data, funnel_element) {
+	// something's wrong, so stop
+	if (!element || !data) return;
```

Comment:
Is `element` meant to be `funnel_element`  ?

Reply:
Just read again the file, `element` is being declared in the global scope, and at the same time being passed as the function's param named `funnel_element`.

Reply:
Indeed; the initial prototype wasn't in a function, and I forgot to fix this. 😅 b9fea3235d